contracts/astroport-tf-liquid-pooler/src/error.rs

```diff
@@ -0,0 +1,54 @@
+use cosmwasm_std::{DecimalRangeExceeded, OverflowError, StdError};
+use neutron_sdk::NeutronError;
+use thiserror::Error;
+
+#[derive(Error, Debug)]
+pub enum ContractError {
+    #[error("{0}")]
+    Std(#[from] StdError),
+
+    #[error(transparent)]
+    NeutronError(#[from] NeutronError),
+
+    #[error(transparent)]
+    OverflowError(#[from] OverflowError),
+
+    #[error(transparent)]
+    DecimalRangeExceeded(#[from] DecimalRangeExceeded),
+
+    #[error("Not clock")]
+    ClockVerificationError {},
+
+    #[error("Single side LP limit exceeded")]
+    SingleSideLpLimitError {},
+
+    #[error("Non zero balances for single side liquidity")]
```

Comment:
nit: this one is not being used, it can be removed. In fact it's not being used in the other LP either